test/e2e/scrapeconfig_test.go

```diff
@@ -491,559 +491,578 @@ func testScrapeConfigDNSSDConfig(t *testing.T) {
 	require.NoError(t, err)
 }
 
+type scrapeCRDTestCase struct {
+	name             string
+	scrapeConfigSpec monitoringv1alpha1.ScrapeConfigSpec
+	expectedError    bool
+}
+
 func testScrapeConfigCRDValidations(t *testing.T) {
 	t.Parallel()
-	name := "test"
+	t.Run("KubernetesSD_API_Validations", func(t *testing.T) {
```

Comment:
(suggestion) we can probably shorten a bit
```suggestion
	t.Run("KubernetesSD", func(t *testing.T) {
```